sdcflows/utils/tools.py

```diff
@@ -23,6 +23,70 @@
 """Image processing tools."""
 
 
+def deoblique_and_zooms(in_reference, oblique, factor=4, padding=1):
```

Comment:
may as well type it since you made that nice docstring
```suggestion
import nibabel as nb

def deoblique_and_zooms(
    in_reference: nb.spatialimages.SpatialImage, 
    oblique: nb.spatialimages.SpatialImage, 
    factor: int = 4, 
    padding: int = 1
):
```

Reply:
Note that `in_reference.header.get_qform()` is used, so `in_reference` needs to be `nb.Nifti1Image`, in case we ever want to type check this.